expan/core/experiment.py

```diff
@@ -20,396 +24,103 @@
 # TODO: add filtering functionality: we should be able to operate on this
 # class to exclude data points, and save all these operations in a log that then
 # is preserved in all results.
-class Experiment(ExperimentData):
+class Experiment(object):
     """
     Class which adds the analysis functions to experimental data.
     """
-    # TODO: add a constructor that takes an ExperimentData!
-    def __init__(self, baseline_variant, metrics_or_kpis, metadata={}, features='default'):
-        # Call constructor of super class
-        super(Experiment, self).__init__(metrics_or_kpis, metadata, features)
-
-        # If no baseline variant is found
-        if ((baseline_variant not in self.kpis.index.levels[
-            self.primary_indices.index('variant')])
-            and (baseline_variant not in self.features.index.levels[
-                self.primary_indices.index('variant')])):
-            raise KeyError('baseline_variant ({}) not present in KPIs or features.'.format(
-                baseline_variant))
-        # Add baseline to metadata
-        self.metadata['baseline_variant'] = baseline_variant
-
-
-    @property
-    def baseline_variant(self):
-        """
-        Returns the baseline variant.
+    def __init__(self, controlVariantName, data, metadata, reportKpiNames=None, derivedKpis=[]):
+        reportKpiNames = reportKpiNames or getColumnNamesByType(data, np.float64)
```

Comment:
1) yes, that was my first solution, maybe include `int` columns too, we can discuss this

2) there are no features yet. I'm not sure that I'm convinced to store features in a different manner than KPIs.